i18n/en.pot

```diff
@@ -188,6 +188,111 @@ msgstr ""
 msgid "Select all"
 msgstr ""
 
+msgid "Filter by name"
+msgstr ""
+
+msgid "Open"
+msgstr ""
+
+msgid "Couldn't load items"
+msgstr ""
+
+msgid ""
+"There was a problem loading items. Try again or contact your system "
+"administrator."
+msgstr ""
+
+msgid "No items found. Create a new to get started."
+msgstr ""
+
+msgid ""
+"No items found. Try adjusting your search or filter options to find what "
+"you're looking for."
+msgstr ""
+
+msgid "Create new"
+msgstr ""
+
+msgid "Open a visualization"
+msgstr ""
+
+msgid "Loading visualizations"
+msgstr ""
+
+msgid "Couldn't load visualizations"
+msgstr ""
+
+msgid ""
+"There was a problem loading visualizations. Try again or contact your "
+"system administrator."
+msgstr ""
+
+msgid "No visualizations found. Click New visualization to get started."
+msgstr ""
+
+msgid ""
+"No visualizations found. Try adjusting your search or filter options to "
+"find what you're looking for."
+msgstr ""
+
+msgid "New visualization"
+msgstr ""
+
+msgid "Open a map"
+msgstr ""
+
+msgid "Loading maps"
+msgstr ""
+
+msgid "Couldn't load maps"
+msgstr ""
+
+msgid ""
+"There was a problem loading maps. Try again or contact your system "
+"administrator."
+msgstr ""
+
+msgid "No maps found. Click New map to get started."
+msgstr ""
+
+msgid ""
+"No maps found. Try adjusting your search or filter options to find what "
+"you're looking for."
+msgstr ""
+
+msgid "New map"
+msgstr ""
+
+msgid "Created"
+msgstr ""
+
+msgid "Last updated"
+msgstr ""
+
+msgid "Type"
+msgstr ""
+
+msgid "Clear filters"
+msgstr ""
+
+msgid "Created by"
+msgstr ""
+
+msgid "Anyone"
+msgstr ""
+
+msgid "Only you"
+msgstr ""
+
+msgid "Others"
+msgstr ""
+
+msgid "{{firstItem}}-{{lastItem}} of {{total}}"
```

Comment:
maybe this string can contain something that indicates that its the count / number of items? looks a bit hard to translate as it is right now? 
e.g. `{{firstIndex}}-{{lastIndex}} of {{totalNumberOfItems}}`

Reply:
This comes from `ui` is not controllable by us.
See: https://github.com/dhis2/ui/blob/master/packages/widgets/src/Pagination/PageSummary.js

Reply:
No, it comes from https://github.com/dhis2/analytics/pull/919/files#diff-f8348f51e29e17d5516c6056d1e558539a04771b7598a2da0b75bdd222ddad5fR17. 
The props comes from the `interpolationObject` yes, but you can change the names of those props before you pass them in to i18n. 
E.g.
```
pageSummaryText={interpolationObject =>
    i18n.t(
         '{{propA}}-{{propB}} of {{propC}}',
        {
            propA: interpolationObject.firstItem,
            propB: interpolationObject.lastItem,
            propC: interpolationObject.total
        }
    )
}
```

To clarify, my point was to make it easier for the translators. So I was just suggesting to change the prop names in the translation string (that ends up in en.pot), not in the code necessarily.

Reply:
Alt, you could even do a named destructuring of `interpolationObject` directly, like `pageSummaryText = ({ firstItem: propA, lastItem: propB, total: propC })`

Reply:
Indeed, we can rename those placeholders 👍

Reply:
Renamed the placeholders.